scripts/dockers/cb_master/configure-portal.py

```diff
@@ -254,10 +255,25 @@ def patch_mail_client(self):
             mailclienthrd.set('instance.smtp.%s' % key, value)
         mailclienthrd.save()
 
+
+    def configure_manifest(self):
+        with open('/opt/cfg/system/versions-manifest.yaml') as file_discriptor:
+            data_str = file_discriptor.read()
+            data_obj = yaml.load(data_str)
+        if self.scl.searchOne({'name': data_obj['version']}):
+            version = self.scl.searchOne({'name': data_obj['version']})
+        version = self.scl.version.new()
```

Comment:
this shoudl be and else block then...
also doesnt version have a timestamp to indicate when this was installed?